client/my-sites/checkout/checkout/paypal-payment-box.jsx

```diff
@@ -123,26 +129,47 @@ export class PaypalPaymentBox extends React.Component {
 	};
 
 	renderButtonText = () => {
+		const testSealsCopy = 'variant' === abtest( 'checkoutSealsCopyBundle' ),
+			renewalText = testSealsCopy
+				? this.props.translate( 'Complete subscription purchase with PayPal', {
+						context: 'Pay button on /checkout',
+				  } )
+				: this.props.translate( 'Purchase %(price)s subscription with PayPal', {
+						args: { price: this.props.cart.total_cost_display },
+						context: 'Pay button on /checkout',
+				  } ),
+			purchaseText = testSealsCopy
+				? this.props.translate( 'Complete purchase with PayPal', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?